### DISCUSSION TITLE
Welcome to E7-RTA-Helper Discussions!

### DISCUSSION THREAD

**Discussion by SamTheCoder777** [SamTheCoder777/E7-RTA-Helper#2] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

**Comment by hp00203722** [comment]
bro，can i use chinese in epic7?

**Reply by SamTheCoder777** [reply]
I am not sure what you mean by using Chinese in epic 7. 

If you are asking a question for the game "Epic 7" you should contact their customer service. 

If you are asking that if this program can be used with Epic 7 on Chinese language settings, the answer is yes.

**Comment by hp00203722** [comment]
Seungmook Han ***@***.***>於 2024年11月23日 週六，上午7:37寫道：

> I am not sure what you mean by using Chinese in epic 7.
>
> If you are asking a question for the game "Epic 7" you should contact
> their customer service.
>
> If you are asking that if this program can be used with Epic 7 on Chinese
> language settings, the answer is yes.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/SamTheCoder777/E7-RTA-Helper/discussions/2#discussioncomment-11354157>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AYYPQBCSYVYFDTO4XPGQAGT2B6527AVCNFSM6AAAAABN5XMX3SVHI2DSMVQWIX3LMV43URDJONRXK43TNFXW4Q3PNVWWK3TUHMYTCMZVGQYTKNY>
> .
> You are receiving this because you commented.Message ID:
> ***@***.***
> com>
>
Yes! I means this program whether can use epic7 on Chinese language~
Thank you

**Comment by hp00203722** [comment]
hope that one day AI combat assistants can be added~

**Comment by jsk373** [comment]
First of all, I’d like to apologize in advance if my English is not smooth, as I’m using a translation tool. I have a few questions I’d like to ask:

#### 1. Data Collection Target
Who were the users targeted for data collection?  
(For example, users in the Legend to Champion tiers.)  
I checked the code but couldn’t find anything related to data collection, so I couldn’t figure out the criteria for the data collection targets.

#### 2. Action Gauge Impact in Epic Seven RTA
Epic Seven RTA is a game where the action gauge has a significant impact.  
When I checked the training data, only the following were recorded:
- Pick order
- First-pick status
- Win/loss outcome  

I’m not sure about the current data collection algorithm, but the official website (epic7.gg) provides action gauge information. Including this data might lead to more accurate results.

Thank you for your attention!

**Reply by SamTheCoder777** [reply]
1. For the data, I am using the top 100 players from each server. (https://epic7.gg.onstove.com/en/rank/server) 

2. My model is trained for recognizing the pattern of the drafting phase instead of trying to understand the game play itself. 

Of course several other factors are important to winning in epic seven such as the stats of each characters, combat readiness, play style, artifacts, equipment, etc. However attempting to make a model that understands all of these factors are a very difficult task and just inputting these data into the model will only confuse it.

So my current model only tries to mimic the draft style of the top players (focusing on the successful ones by collecting win/loss). Recognizing the patterns such as which characters are picked with and which are picked to counter an enemy's pick (with the pick order/ first-pick status).
